migrations/20150224_force_resting_in_inn.js

```diff
@@ -0,0 +1,58 @@
+var migrationName = '20150224_force_resting_in_inn';
+var authorName = 'Alys'; // in case script author needs to know when their ...
+var authorUuid = 'd904bd62-da08-416b-a816-ba797c9ee265'; //... own data is done
+
+/*
+ * force all active players to rest in the inn due to massive server fail
+ */
+
+var dbserver = 'localhost:27017' // CHANGE THIS FOR PRODUCTION DATABASE
+
+var mongo = require('mongoskin');
+var _ = require('lodash');
+
+var dbUsers = mongo.db(dbserver + '/habitrpg?auto_reconnect').collection('users');
+
+var query = {
+    'auth.timestamps.loggedin':{$gt:new Date('2015-02-22')}
```

Comment:
Yes, for future iterations of the script. It would also be worth adding a filter to exclude players who aren't already sleeping, to speed it up a little.